src/coreclr/nativeaot/Test.CoreLib/src/Test.CoreLib.csproj

```diff
@@ -22,8 +22,6 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(InPlaceRuntime)' == 'true'">
     <DefineConstants>INPLACE_RUNTIME;$(DefineConstants)</DefineConstants>
-    <DefineConstants Condition="'$(Platform)' == 'arm'">FEATURE_64BIT_ALIGNMENT;$(DefineConstants)</DefineConstants>
```

Comment:
On line 10, we can use `Condition="'$(Is64Bit)' != 'true'"` and delete redundancies (same in `System.Private.CoreLib.csproj`).

Reply:
It's not set to `true` for x86. We may want to set it for x86 but then we have a disparity between `FEATURE_64BIT_ALIGNMENT` and `FEATURE_DOUBLE_ALIGNMENT_HINT` on the native side.

Reply:
Ah, I thought 458bfe72f74e3cde66b200662dcce26a031d7427 is enabling it for x86.

Reply:
I still have to come up with a strategy how to enable the NativeAOT support for `FEATURE_DOUBLE_ALIGNMENT_HINT`. Technically I probably want the same code paths as `FEATURE_64BIT_ALIGNMENT` on the managed side but I would find it confusing to mix the two #defines.

Reply:
> how to enable the NativeAOT support for FEATURE_DOUBLE_ALIGNMENT_HINT.

NativeAOT does not have support for FEATURE_DOUBLE_ALIGNMENT_HINT today. I would not do it as part of this PR if ever. It was perf micro-optimizations for numerical algorithms on win-x86. We may want to go in opposite direction - measure what kind of benefit it provides on current hardware and delete it if the benefit is negligible.